### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-03-27

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#676]
## Time

**UTC Wed 27-Mar-2019 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 27-Mar-2019 09:00 (09:00 AM) |
| US / Mountain | Wed 27-Mar-2019 10:00 (10:00 AM) |
| US / Central  | Wed 27-Mar-2019 11:00 (11:00 AM) |
| US / Eastern  | Wed 27-Mar-2019 12:00 (12:00 PM) |
| London        | Wed 27-Mar-2019 16:00 (04:00 PM) |
| Amsterdam     | Wed 27-Mar-2019 17:00 (05:00 PM) |
| Moscow        | Wed 27-Mar-2019 19:00 (07:00 PM) |
| Chennai       | Wed 27-Mar-2019 21:30 (09:30 PM) |
| Hangzhou      | Thu 28-Mar-2019 00:00 (12:00 AM) |
| Tokyo         | Thu 28-Mar-2019 01:00 (01:00 AM) |
| Sydney        | Thu 28-Mar-2019 03:00 (03:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2019-03-27&iso=20190327T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Mar+27%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1nrKjgchNNalkYWpUwLwSLLzdNHX5WR7A7i0coqPQiWU/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/Release

* Preparation for Node.js 12  [#417](https://github.com/nodejs/Release/issues/417)

### nodejs/summit

* Call for Sessions: Berlin Collab Summit [#149](https://github.com/nodejs/summit/issues/149)

### nodejs/TSC

* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)

### nodejs/admin

* Checklist to address changes due to move to OpenJS Foundation [#311](https://github.com/nodejs/admin/issues/311)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Ali Ijaz Sheikh @ofrobots (TSC)
* Rod Vagg @rvagg (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Rich Trott @Trott (TSC)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Trott** [comment]
https://github.com/nodejs/Release/issues/417 and https://github.com/nodejs/summit/issues/149 were added for visibility/awareness only so they can be covered quickly as part of the Announcements section.

https://github.com/nodejs/admin/issues/311 probably requires some more detailed conversation. Although if that conversation could happen in the issue tracker, that would be preferable IMO. And while that stuff is important and should not be ignored, it is likely not yet urgent?

In short, there may be grounds for canceling the meeting. (Personally, I'd hold off and only cancel if there's an uptick in action on https://github.com/nodejs/admin/issues/311, which could be taken as an indication that the conversation is moving forward without the need for a meeting.)

**Comment by rvagg** [comment]
Re Node 12, more attention and careful thought for supported platforms would be good, still. https://github.com/nodejs/node/pull/26714 (not ready for merge but nearing it)

* We're settling on GCC-6 where GCC is in use.
* Almost no feedback on macOS defaults, proposal is to lift minimum supported compile OS to 10.13 and Xcode 10 with minimum supported run OS to 10.11.
* Python 2/3 is awkward, we're probably going to have to recommend having both out of the gate but with 2 EOL at the end of the year we have to be careful to not lock ourselves into terrible recommendations.

**Comment by mhdawson** [comment]
I'm out on vacation this week.  @mcollina, @danbev, @Trott any chance one of you can chair?

**Comment by mhdawson** [comment]
In respect to nodejs/admin#311, one of the important issues to weight in on is: https://github.com/nodejs/admin/issues/316.  If the meeting is held we should discuss 316 to see if there  is a consensus of the TSC as to what we would prefere.

**Comment by Trott** [comment]
> I'm out on vacation this week. @mcollina, @danbev, @Trott any chance one of you can chair?

Did you mean to perhaps include @fhinkel?

I can _probably_ chair, although (a) happy if someone else does it (especially because I'm a "probably" and not a "definitely"), and (b) I don't have the elevated privileges or password or whatever is needed for Zoom....

**Comment by Trott** [comment]
> one of the important issues to weigh in on is: nodejs/admin#316. If the meeting is held we should discuss nodejs/admin#316 to see if there is a consensus of the TSC as to what we would prefer.

So far, there are four TSC members all agreeing to the current process/proposal and no one objecting. I'd definitely encourage people to weigh in on that issue rather than hashing it out at the meeting.

**Comment by danbev** [comment]
@mhdawson I would have been happy to but this is a meeting time that I can't make.

**Comment by Trott** [comment]
On nodejs/admin#316, we're up to 8 TSC members endorsing Myles's _status quo_ proposal and 1 TSC member suggesting a variation (but not appearing to have a problem with _status quo_). If we do decide to discuss it at the meeting still, the discussion may be a short one.

**Comment by Trott** [comment]
Hey, @nodejs/tsc! With @BethGriggs having announced the @nodejs/release semver-major cutoff date for Node.js 12, it's a good idea to go through the list off open PRs labeled `semver-major` and see if there's anything in there you might want to push along so it can get into 12.x!

https://github.com/nodejs/node/search?q=is%3Apr+is%3Aopen+label%3Asemver-major&type=Issues

OMG, there are **32** pull requests that are open and labeled semver-major. Let's get reviewing! (Probably a good idea to close ones that are unlikely to ever land.)

**Comment by cjihrig** [comment]
Am I the only one that is a little uneasy seeing a few semver major PRs to the module system (especially in light of issues like https://github.com/nodejs/node/issues/26926)?

**Comment by MylesBorins** [comment]
I'd like to see https://github.com/nodejs/node/pull/26745 added to the agenda for a heads up at the very least. Planning to land tomorrow if there are no objections.

I may not be able to attend the meeting due to TC39 but could likely step out for a specific agenda item if y'all ping me when you need me

**Comment by Trott** [comment]
Minutes PR: https://github.com/nodejs/TSC/pull/679

**Comment by thefourtheye** [comment]
No significant moderation team activity last week.

---

cc @nodejs/tsc @nodejs/community-committee @nodejs/moderation
